spec/forms/candidate_interface/volunteering_role_form_feature_flag_on_spec.rb

```diff
@@ -0,0 +1,196 @@
+require 'rails_helper'
+
+RSpec.describe CandidateInterface::VolunteeringRoleForm, type: :model do
+  FeatureFlag.activate('restructured_work_history')
```

Comment:
Wondering if this needs to be in a `before` block?
Would having it here cause state to leak out into other specs and cause flakiness?